dotCMS/src/main/java/com/dotmarketing/exception/DotCorruptedDataException.java

```diff
@@ -0,0 +1,20 @@
+package com.dotmarketing.exception;
+
+/**
+ * Exception to report errores when the data is corrupted on some storage
+ * @author jsanca
+ */
+public class DotCorruptedDataException extends DotRuntimeException {
+
+    public DotCorruptedDataException(String message) {
+        super(message);
+    }
+
+    public DotCorruptedDataException(Throwable cause) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'cause' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5547873)